src/Engine/System/RenderSystem.hpp

```diff
@@ -9,6 +9,7 @@
 
 #include <map>
 #include <string>
+#include <iostream>
```

Comment:
iostream shouldn't be included in headers

Reply:
Removing this line gives "'cerr' is not a member of 'std". Where should we place this line?